third_party/move/move-compiler-v2/tests/checking/attributes/aptos_stdlib_attributes.move

```diff
@@ -0,0 +1,10 @@
+// Test that warnings about unknown "#[testonly]" attribute is
```

Comment:
I am assuming this comment is here (although a bit confusing to the reader at first) is to maintain the same contents of the same-named file in the `skip_attribute_checks` section?

On a related note, is there some expectation to maintain some sort of parity between the move files in this folder and the one in `skip_attribute_checks`? I see that there is currently a partial overlap.

Reply:
There wasn't an intention of parity.

checking/attributes is some select tests from v1 (all from move_check/parser, as it happens) that had attribute tests so that I could see what would happen here.  this directory didn't exist in v1, but I needed some particular tests here so I added a directory to group them.

v1's skip_attribute_checks was originally just some selected tests containing attributes to make sure that unknown attributes were *not* flagged if --skip-attribute-checks was enabled.  this directory was moved from v1 in its entirety.